BinaryEncoding.md

```diff
@@ -439,28 +439,25 @@ It is legal to have several entries with the same type.
 | Name | Opcode | Immediates | Description |
 | ---- | ---- | ---- | ---- |
 | `unreachable` | `0x00` | | trap immediately |
-| `block` | `0x01` |  | begin a sequence of expressions, the last of which yields a value |
-| `loop` | `0x02` |  | begin a block which can also form control flow loops |
-| `if` | `0x03` | | begin if expression |
+| `block` | `0x01` | arity : `varuint1` | begin a sequence of expressions, yielding 0 or 1 values |
+| `loop` | `0x02` | arity : `varuint1` | begin a block which can form control flow loops |
```

Comment:
Alternatively, I still like the symmetry of giving branch target immediates a direction bit; in that case the arity of `loop` would be meaningful in the same way as `block`.